### ISSUE TITLE
Как настроить или скомпилировать под другие драйвера?

### ISSUE THREAD

**Issue by chshkhr** [vyacheslav-shubin/shui#1]
Здравствуйте, долго и мучительно боролся с оригинальными прошивками от MKS и сегодня случайно наткнулся на ваш форк. Начал тестировать. Просто восторг! Браво! Интересно было бы попробовать также Shui на моих Bluer и SapphirePro. Но у меня другие драйвера. Есть варианты?

**Comment by vyacheslav-shubin** [comment]
https://t.me/joinchat/8eQiBVI29zw1YTU6

**Comment by chshkhr** [comment]
Спасибо. Буду следить. Я тогда пока лучше на sh1.0.0 посижу, которая с исходниками. Как компилировать их я не понял, но в папке snapshot есть готовая прошивка, которая мне и понравилась. А новые из этого репозитория не работают на обоих принтерах. В левом верхнем углу вместо координат вопросики и на команды перемещения и домой никак не реагирует. Перестановки драйверов, чтобы соответствовать cinfiguration.h, и перепрошивки разных версий не помогают... Удачи и спасибо!

**Comment by vyacheslav-shubin** [comment]
Вероятно, вы не запустили файл конфигурации "на печать"

**Comment by chshkhr** [comment]
Да, наверное, поэтому. Попробую. Я таки скомпилировал sh1.1 прошивку. В С/С++ я не особо разбираюсь, но чтобы подправить порядок кнопок ManualLevelingUI и отбражение картинок MoreUI (отбражалась последняя везде) квалификации хватило. Эта версия уже устарела, но, возможно, правка в MoreUI пригодится на будущее, так что если надо, сделаю Pull request

**Comment by chshkhr** [comment]
> 
> 
> Вероятно, вы не запустили файл конфигурации "на печать"

Возможно стоит сделать автозагрузку **необходимых** для работы файлов конфигурации

**Comment by vyacheslav-shubin** [comment]
Можно и сделать... Всему своё время ;).

**Comment by SkifBor** [comment]
Добрый день! Можно ли накать Вашу прошивку на MKS TINYBEE?

**Comment by vyacheslav-shubin** [comment]
Нет
